src/blockchain/chain.rs

```diff
@@ -35,6 +35,7 @@ pub struct BlockForHistory {
 }
 
 impl Chain {
+    #[cfg_attr(test, allow(dead_code))]
```

Comment:
It's just because of a warning caused by the manual mock of `start_blockchain_for_new_bill`, which makes this unused in `cfg(test)`.

Once the Blockchain code is behind a nice abstraction and can be more comfortably mocked for tests, this will go away.